templates/components/panels/get-involved.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="w-100 mw-none ph3 mw8-m mw9-l center f3">
     <header>
       <h2>Get involved</h2>
-      <div class="highlight">
+      <div class="highlight"></div>
     </header>
       <div class="row media">
```

Comment:
Hm, we should probably change the indentation here too to prevent any future regressions. Would you mind moving this section one to the left so the indentation lines up again?

Reply:
Yep done in https://github.com/rust-lang/www.rust-lang.org/pull/668/commits/1b1187618dc3dccfea20b8046c832a28d2811e61